src/config/config.ts

```diff
@@ -3,6 +3,21 @@ const cloudflareIfps = process.env.REACT_APP_CLOUDFLARE_IFPS ?? 'https://cloudfl
 // eslint-disable-next-line no-console
 console.log(process.env.REACT_APP_NETWORK_ID, process.env.REACT_APP_CONTRACT_ID);
 
+const SUPPORTED_WALLET_OPTIONS = [
+  {
+    symbol: 'MYNEAR',
```

Comment:
What is symbol and wallet name used for here? Shouldn't we have an icon field and human readable name? For example, the ID is `mynearwallet` but the name is `My NEAR Wallet` which is what is rendered to the user here:
<img width="226" alt="image" src="https://user-images.githubusercontent.com/57506486/222243990-6c5acccf-31a4-4292-b43b-a9771cef8c8e.png">

Reply:
yeah i agree, its kinda confusing so i will refactor it